content/wiki/long-term-roadmap.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "Post-July Linebender roadmap"
++++
+
+Unstructured notes from the RustNL roadmap brainstorming that were left out of the May-July roadmap blog post.
+
+This may become a living document, eg a long-term roadmap for the project. For now it's the post-July roadmap.
+
+## Vello
+
+* Emoji
+* Image filters / blur / shadows
+    * Render graph (Raph to drive, see [thread](https://xi.zulipchat.com/#narrow/stream/197075-gpu/topic/Render.20Graph))
+* Gpu embedding / wgpu engine abstraction (see [Servo thread](https://xi.zulipchat.com/#narrow/stream/147922-new-members/topic/Greg.20from.20Servo))
+* Glyph caching
+    * Render graph? 
+    * or CPU side? 
+    * (sparse strips probably)
+* Frame pacing
+* Damage regions
+    * MVP: just render sub-rects
+* CPU impl
+    * Testing
+    * Downlevel
+* Fuzzing?
+
+## Masonry
+
+* ‘Compositing’
+* Frame pacing (relevant to Vello and AccessKit)
+* Damage regions
+* Styling options
+* Copy features from [Tomorrow Corporation Tech Demo](https://www.youtube.com/watch?v=72y2EC5fkcE)
+
+## Parley
+
+* IME support
+    * Suggestion regions?
+* Split out blob & other vocabulary types
+    * Blob may be prioritized by other project needs
+
+## Xilem
+
+* Android
+    * Material UI style widgets/views? Requires styling
+* Styling
+* Hot reloading
+* Async behavior in Masonry widgets (image loading etc)
+* Cleanup run for issues and PRs
+
+## AccessKit
+
+* Haven’t talked about it a lot
+  * Improve documentation (see [#402](https://github.com/AccessKit/accesskit/issues/402))
+  * Point to / create a repository of accessibility best practices.
+  * Not just code, also social: Have people with disabilities test your stuff!
+  * Point to resources, forums, etc? Need to talk to Matt about this. (What were the best practices at Microsoft?)
+  * Accessibility isn’t just screen readers. Also color contrast, customizing size, IME support, motor disabilities (avoid stuff that requires simultaneous input).
+
+* “Accessibility is more than adding a dependency to AccessKit”
+    * (Olivier: Note to self: include GMTK video about Celeste Assist Mode)
```

Comment:
Does this need to be handled?

Reply:
Maybe if I get around to writing an article about accessibility. In the meantime, those are unstructured notes.